dame.html

```diff
@@ -0,0 +1,23 @@
+<section class="spade">
+  <h1>Dame</h1>
+  <var>♠</var>
+  <hr>
+</section>
```

Comment:
section - used for big content boxes, like big parts of application. On this case, we can use div
h1 - always uses 1 time, like a title/logo text of web application
hr - supermegaold and antisemantic tag in html.